src/main/java/com/alkemy/ong/entity/CommentEntity.java

```diff
@@ -0,0 +1,47 @@
+package com.alkemy.ong.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.SQLDelete;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "COMMENTS")
+@SQLDelete(sql = "UPDATE users SET soft_delete = true WHERE id=?")
```

Comment:
Do we add the @Where(clause = "soft_delete=false") ?

Reply:
For the list of all the Comments it would be useful